bin/ebs-autoscale

```diff
@@ -140,7 +143,9 @@ add_space () {
       else
         logthis "Adding device ${DEVICE} to logical volume /dev/mapper/${LVM_VG}-${LVM_LV}"
         vgextend ${LVM_VG} ${DEVICE}
-        lvresize -l 100%VG /dev/mapper/${LVM_VG}-${LVM_LV}
+        while ! timeout 5 lvresize -l 100%VG /dev/mapper/${LVM_VG}-${LVM_LV}; do
+          sleep 1
+        done
```

Comment:
does this have a chance to go in a infinite loop? from my experience i think we should always cap such loops from now on